config/locales/idv/en.yml

```diff
@@ -133,7 +133,7 @@ en:
       clear_and_start_over: Clear my information and start over
       come_back_later_html: <p><strong>Letters typically take 3 to 7 business days to
         arrive</strong><br /> through USPS First Class Mail.</p> <p>Once your
```

Comment:
This is one of only two <br> tags in the whole file, and I think the other one might also have been added for this view. Do we want those in here when people will be viewing the app on different width screens?

Reply:
I think we should ask the design team. From the Figmas it looks like there is a perhaps intentional separation of the first, bolded line with an explicit break

Reply:
I talked to the design team and we are going forward with just bolding the `3 to 7 business days` part and removing the line break.